comp/core/status/component.go

```diff
@@ -20,7 +20,7 @@ const CollectorSection string = "collector"
 // Component interface to access the agent status.
 type Component interface {
 	// Returns all the agent status information for the format type
-	GetStatus(format string, verbose bool) ([]byte, error)
+	GetStatus(format string, verbose bool, excludeSection ...string) ([]byte, error)
```

Comment:
In the GUI section we need to fetch all the status except for the collector section. I was debating between adding a specific method to the interface `GetStatusExcept(format string, verbose bool, excludeSection ...string)` in a way I felt that would be more appropriate, but first I wanted to hear someone opinions about that.

Reply:
What do you think about adding a new type derived from string  for `excludeSection ...string`? It would allow to have the list of possible values. 
```go
type Section string

const section1 = Section("Section 1")
const section2 = Section("Section 2")
```
Another option is to use [iota](https://yourbasic.org/golang/iota/).

Reply:
> In the GUI section we need to fetch all the status except for the collector section. I was debating between adding a specific method to the interface `GetStatusExcept(format string, verbose bool, excludeSection ...string)` or modify the `GetStatus` method. In a way, I felt that would be more appropriate, to add the new method `GetStatusExcept` but first, I wanted to hear someone's opinions about that.

For another PR, but could we remove this whole behavior from the GUI and simply show the entire status page instead of splitting it int two ? I'm not sure why we did this in the first place but I'm pretty sure it's some legacy behavior.

Reply:
> For another PR, but could we remove this whole behavior from the GUI and simply show the entire status page instead of splitting it int two ? I'm not sure why we did this in the first place but I'm pretty sure it's some legacy behavior.

I'm very happy about doing that. That way we can get rid of this `excludeSection` thingy

Reply:
> I think section are dynamic based on what providers are registered to avoid having a hard-coded list of section,

That is correct